File: core/src/main/java/de/qabel/core/config/factory/ContactFactory.java
package de.qabel.core.config.factory;

import java.util.Collection;

import de.qabel.core.config.Contact;
import de.qabel.core.crypto.QblECPublicKey;
import de.qabel.core.drop.DropURL;

public interface ContactFactory {
    Contact createContact(QblECPublicKey publicKey, Collection<DropURL> dropUrls, String alias);
}

File: core/src/main/java/de/qabel/core/config/factory/ContactFactory.kt
package de.qabel.core.config.factory

import de.qabel.core.config.Contact
import de.qabel.core.crypto.QblECPublicKey
import de.qabel.core.drop.DropURL

interface ContactFactory {
    fun createContact(publicKey: QblECPublicKey, dropUrls: Collection<DropURL>, alias: String): Contact
}
